### DISCUSSION TITLE
Questions about pricing

### DISCUSSION THREAD

**Discussion by mstade** [supabase/supabase#1173] [Questions] [answered]
Hi!

I have a few questions regarding the recent announcements with regards to supabase pricing. Before getting on with it – massive kudos on the big updates, really happy to see all the great progress you're making!

The free tier is generous, which is great for getting started and not worrying about costs when just trying to get a project off the ground. The Pro tier is very generous, at $25 / month it's a no brainer even for projects with modest of runways. The PAYG tier is where things start to get confusing to me. In the email you sent out the pricing TL;DR was:

> Pricing - Our most frequently asked question by far is "ok supabase is sweet, but how much is it going to cost?". TL;DR there's Free Tier up to 500mb + 10k auth users, a Pro Tier at $25/month for 8GB + 100k auth users, and **anything additional** is charged on a usage basis.

That last bit which I've emphasized doesn't seem to match what's on the [pricing page](https://supabase.io/pricing), or I'm just confused by the latter. The above statement suggests that the PAYG tier is the same as Pro, except for when you reach usage levels above what's given in the Pro tier, but I can't see this reflected on the pricing page. Is it correct to assume that PAYG means Pro + extra usage costs, or is PAYG $25 / month + usage starting from zero? So to match the Pro tier in terms of storage and transfer limits, the actual cost for PAYG is aproximatly $42 / month?

Also, when the alpha came to a close and [the beta was announced](https://supabase.io/beta), the following commitments were made:

> - All Alpha Users will receive credits equivalent of 2 years of base tier usage. These will automatically be credited to your account if you signed up prior to December 2020.
> - All Beta Users (new users from December 2020) will receive 1 year of base tier usage for free.
> University (and participating code school) Students will be eligible for 2 years of base tier usage (Code Schools can contact rory@supabase.io)
> - Early stage startups participating in selected incubator programs can claim additional credits which can be applied to products outside of the base tier.
>
> The Supabase Base Tier is now called the Supabase Pro tier as per the pricing page: https://supabase.io/pricing

If I recall correctly we signed up during the tail end of the alpha (late november) so presumably we are eligible for the 2 year Pro tier credit, but when does this come into effect? Has that already started or does it come into effect when we sign up for the Pro or PAYG tier? I can't see anything in the dashboard to suggest there's credit available.

Finally, how do you sign up for PAYG? I can only see "Upgrade to Pro" in the dashboard, do you first have to upgrade to Pro in order to go to the PAYG tier? (For the record, we won't need PAYG just yet, just curious.)

**Comment by kiwicopple** [comment, marked as answer]
Thanks for the questions @mstade. Also before I begin it might be helpful to point out this blog post for anyone who missed it: https://supabase.io/blog/2021/03/29/pricing

> Is it correct to assume that PAYG means Pro + extra usage costs, or is PAYG $25 / month + usage starting from zero?

As with our docs, it seems we could improve our pricing page a lot. It's hard to get everything right first time, so we'll adjust the language on the pricing page to make this clearer:

PAYG is $25 / month + usage starting from zero. e.g.: it's not "overage"

> presumably we are eligible for the 2 year Pro tier credit, but when does this come into effect

If you are an early sign up, please reach out to beta@supabase.io and we'll add the credits to your account. When we announced the Credits scheme, we weren't planning to offer a free tier at all, so just take note that the credits are also on top of the Free Tier :)

> Finally, how do you sign up for PAYG?

Also reach out to beta@supabase.io for now. As with everything we're doing, we shipped billing very early so we haven't finished the "self-serve usage" yet. We should have it finished in a month or so

**Reply by mstade** [reply]
Thanks @kiwicopple, much appreciated – we'll definitely reach out!

I'd love to provide some constructive criticism on how to improve the pricing page but to be perfectly honest, I don't know how to. There's nothing there to explicitly say that the tiers are additive, but still that's how I initially read it and then got confused. Maybe I've been conditioned by too many services doing exactly that, offering tiers that include whatever the lower tier includes and then some, and laying them out side by side... It's a very real possibility that your pricing info is fine, and I'm just a dunce! 🤪

Speaking of being a dunce, I'm having a bit of trouble parsing your answer about the credits. What does this mean?

> When we announced the Credits scheme, we weren't planning to offer a free tier at all, so just take note that the credits are also on top of the Free Tier :)

I'm not a native english speaker, so that may explain some of my dunceness, I'm sorry! 😅

**Reply by roryw10** [reply]
Hi @mstade - with respect to the credits, when we initially announced the credits scheme we didn't intend to offer a free tier. We thought the credits would give users the ability to try Supabase at zero cost in a pricing structure that had every tier as a paid tier. 

As we progressed our discussions with users, and as developers ourselves, we felt that having a free tier was important and updated our strategy around pricing. Despite the free tier being now part of our pricing strategy, we still want to honour the commitment we made to our early adopters to give credits away to those who signed up early.

This means that alpha and beta users get enough credits for the value of 2 years pro tier usage (alpha users $600 credits) and 1 year pro tier usage (beta users $300 credits) whilst everyone benefits from being able to try Supabase out for free with the free tier. These credits can be used for pro or PAYG tiers. 

In your case, if you email beta@supabase.io we can apply the credits to your account. From there, you can use the free tier as required and when you upgrade to pro then we will then start to deduct from your credits balance on a monthly basis.

**Reply by mstade** [reply]
Brilliant answer that even a dunce like me can understand, many thanks @roryw10! 🙏😊

**Reply by roryw10** [reply]
No worries at all, always happy to get feedback and we will definitely think about how we can make pricing clearer. If you need anything else just reach back out.
